### DISCUSSION TITLE
Images on map

### DISCUSSION THREAD

**Discussion by BigMinnie** [Zrips/CMI#2663] [Feature Requests]
CMI is already a powerfull tool that every server should have but next to holograms, portals, armour stand editor and so on there is still missing one thing and this is to put images on maps. I know that there is already tons of plugin doing that (like bannerboard, g-sign, ImageOnMap) but a) they are not updated or b) they are broken but for CMI it would mean another great future in already amazing plugin.

**Comment by Relaxing9** [comment]
or you can do it this way: https://mc-map.djfun.de/ (once created drag and drop into world data folder follow instructions on page)
That tool is not broken, but it requires a server restart in order to see the results. Using it would be more practical if this was added. #2737 
Also I am running 1.15.2.

**Comment by BigMinnie** [comment]
It's solution but I think the editor would still be the best similar how the plugins I mentioned have.
